templates/node-class.mustache

```diff
@@ -14,6 +14,15 @@ var {{&className}} = (function(){
         this.domain = domain;
     }
 
+    {{#isSecure}}
```

Comment:
@wcandillon saw your second comment after posting mine. Do you want me to add this ability as an option?
It should support backwards compatibility? ->
```javascript
if (typeof domain === 'object') {
    ...
} else if (typeof domain === 'string') {
    ...
}
```
?